common/network-common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -88,7 +88,7 @@
   // Separate thread pool for handling ChunkFetchRequest. This helps to enable throttling
   // max number of TransportServer worker threads that are blocked on writing response
   // of ChunkFetchRequest message back to the client via the underlying channel.
-  private static EventLoopGroup chunkFetchWorkers;
+  private EventLoopGroup chunkFetchWorkers;
```

Comment:
This can be final, though will require explicitly setting it to null in the constructor if not otherwise initialized

Reply:
yes makes sense to make it final thanks will do